Configuration/FlexForms/TeaIndex.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<T3DataStructure>
+    <sheets>
+        <pagesAndFolders>
+            <ROOT>
+                <TCEforms>
+                    <sheetTitle>
+                        LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index.settings.tab.pagesAndFolders
+                    </sheetTitle>
+                </TCEforms>
+                <type>array</type>
+                <el>
+                    <settings.singleViewPageUid>
+                        <TCEforms>
+                            <label>
+                                LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index.settings.pagesAndFolders.singleViewPageUid
+                            </label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
+                                <allowed>pages</allowed>
+                                <size>1</size>
+                                <minitems>1</minitems>
+                                <maxitems>1</maxitems>
+                                <hideSuggest>1</hideSuggest>
+                            </config>
+                        </TCEforms>
+                    </settings.singleViewPageUid>
+                    <persistence.storagePid>
```

Comment:
Do we have a reason to build the two persistence options ourself, instead of enabling the `tt_content` fields?

Reply:
We discussed this, and will use this approach.
The core supports this, as well as two database fields.
But we prefer the FlexForm, as this bundles everything together in one file, and follows a single concept.